docs/topics/i18n/translation.txt

```diff
@@ -730,6 +730,25 @@ will result in the entry ``"First sentence. Second paragraph."`` in the PO file,
 compared to ``"\n  First sentence.\n  Second sentence.\n"``, if the ``trimmed``
 option had not been specified.
 
+Also ``{% blocktrans %}`` has option `force_escape` to force HTML escaping for
+translated content. It behaves the same as if you applied :tfilter:`force_escape`
+filter, e.g.:
+
+.. code:: html+django
+
+    {% blocktrans force_escape %}
+        It's quick & easy to get up and running with Django.
+    {% endblocktrans %}
+
+
+will produce ``"It&#39;s quick &amp; easy to get up and running with Django."``.
+This option is necessary to make sure that unescaped characters won't break your
```

Comment:
How about giving a practical example that demonstrates this instead of the one above?